src/app/weather-forecast/weatherForecastApiService/weather-forecast-api.service.ts

```diff
@@ -0,0 +1,17 @@
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class WeatherForecastApiService {
+
+    constructor(private http: HttpClient) { }
+
+    private wfApiUrl = 'https://api.openweathermap.org/data/2.5/weather?q=Pisa,it&appid=d5754a8efdf1ead276cffa4f8250f1e1';
+
+    public getOBS() {
```

Comment:
Bad name for this function. This function does not get an observable but the weather data.

Reply:
Make this function parametric on the city